src/app/views/post/_page/page.vue

```diff
@@ -102,7 +102,7 @@
 						-->
 						<div ref="sticker-scroll" />
 
-						<div v-if="post.hasMedia" class="-media-items">
+						<div v-if="post.hasMedia" :key="`media-${post.id}`" class="-media-items">
```

Comment:
I think you should put a comment for this key and the other to say why we need it.